Tests/SwiftFusionTests/Core/VectorNTests.swift.gyb

```diff
@@ -7,7 +7,7 @@ import SwiftFusion
 % import math
 % dims = range(1, 10)
 
-class ConcreteVectorTests: XCTestCase {
+class VectorNTests: XCTestCase {
   % for dim in dims:
```

Comment:
```suggestion
  % for N in vectorSizes:
```

Take or leave these suggestions, but I think this would end up being easier to read.